src/include/miopen/graphapi/enginefinder.hpp

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
```

Comment:
let's rename the file to something better, e.g., `opgraph_descriptor.hpp` to indicate that it implements the `descriptor` behavior. Same with the `.cpp` file.

Reply:
agree